extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -161,3 +214,19 @@ class="jetpack-podcast-player__episode-link"
 
 	return ob_get_clean();
 }
+
+/**
+ * Returns a CSS class based on the context a color is being used and its slug.
+ *
+ * @param string $color_context_name Context/place where color is being used e.g: background, text etc...
+ * @param string $color_slug         Slug of the color.
+ *
+ * @return string String with the class corresponding to the color in the provided context.
+ */
+function get_color_class_name( $color_context_name, $color_slug ) {
+	if ( ! isset( $color_context_name ) || ! isset( $color_slug ) ) {
+		return null;
```

Comment:
In that case, I'll have to change `isset` instead of `empty`. Changing...

Reply:
Huh, I did not realize that. I wonder if that is a bug or feature :) 
Feel free not to change it if it works

Reply:
hahaha, to me it's a feature!

Reply:
Sorry, I seem to have posted that answer to the wrong conversation. I thought I was adding to the variable-in-string conversation. 

I still think we should change this to return an empty string. You don't necessarily have to add an `empty()` check since `''` evaluates as falsy.